### PR TITLE
fix #1125

### LINKED ISSUE #1125: Conservation values have gone bye-bye?
It appears we have lost touch with evolutionary conservation. We wonder where and when?

### PR DESCRIPTION
**Pull request by northwestwitch**
Took me a while to understand that variants could have multiple scores for either GERP, PHAST and PhyloP and that multiple values where tuples..!

### CODE REVIEW

File: scout/constants/variant_tags.py [hunk 1]
```diff
@@ -9,7 +9,20 @@
...
 
-CONSERVATION = ('NotConserved', 'Conserved')
+CONSERVATION = {
+    'dbNSFP_GERP___RS' : {
+        'conserved_min' : 2,
+        'conserved_max' : 10
+    },
+    'dbNSFP_phastCons100way_vertebrate': {
+        'conserved_min' : 0.8,
+        'conserved_max' : 100
```
Comment (dnil) [inline scout/constants/variant_tags.py]: I'll assume you have checked these values, or that they are the same as the previous terms: as far as I remembered, phastCons used to give a P-val? Is it scaled or the -log P perhaps to be more similar to the others?

File: scout/parse/variant/conservation.py [hunk 2]
```diff
@@ -1,47 +1,52 @@
...
             conservations(dict): A dictionary with the conservations
     """
     conservations = {}
-    
+
     conservations['gerp'] = parse_conservation(
-                                            variant, 
-                                            'GERP++_RS_prediction_term'
+                                            variant,
+                                            'dbNSFP_GERP___RS'
```
Comment (dnil) [inline scout/parse/variant/conservation.py]: Weird name, but that's not our doing. 😄

File: scout/constants/variant_tags.py (hunk 1 above)
Comment (northwestwitch) [reply scout/constants/variant_tags.py]: I took these values from here: https://github.com/Clinical-Genomics/MIP/blob/master/templates/rank_model_cmms_-v1.20-.ini

as also @henrikstranneheim  suggested. Do they make any sense to you?

**Comment by northwestwitch** [comment]
Numbers have been there the whole time, strings and numbers have been both there until we updated to the latest MIP version. This code won't break because numbers will be found also for old samples, even if they have the string terms. Your suggestion is good if numeric values have changed over time though.

**Pull request merged**